Tests/SonarQube.TeamBuild.PreProcessor.Tests/RoslynAnalyzerProviderTests.cs

```diff
@@ -88,11 +88,7 @@ public void RoslynConfig_MissingRuleset()
       </AdditionalFile>
     </AdditionalFiles>
   </Configuration>
-  <Deployment>
```

Comment:
The Deployment section isn't required for this test. Omitting it makes that clearer.